cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -55,6 +55,18 @@ export default class SortableTablePo extends ComponentPo {
     return cy.getId('sortable-table-promptRemove');
   }
 
+  deactivateButton() {
+    return cy.getId('sortable-table-deactivate');
+  }
+
+  activateButton() {
```

Comment:
like deactivateButton, this looks list specific

Reply:
Also moved to users.po.ts since its list specific